### DISCUSSION TITLE
Some new XP talent descriptions don't display properly

### DISCUSSION THREAD

**Discussion by DaEncoder** [FakeFishGames/Barotrauma#13813] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image_2024-04-24_100540455](https://github.com/FakeFishGames/Barotrauma/assets/168010934/fead336c-27df-4611-b120-35985bdfe37f)

### Reproduction steps

1. Play as security
2. Check talents tab

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I created a ticket about this in the issue tracker now, we'll look into it! https://github.com/FakeFishGames/Barotrauma/issues/13817

**Comment by bananaman6213** [comment]
I'll add that a number of talents seem broken by this
Health Insurance and Nobody Important Dies have similar looking issues
Bootcamp, Machine Maniac, and Junction Junkie all have incorrect descriptions
![image](https://github.com/FakeFishGames/Barotrauma/assets/119143410/0e939c0e-1d97-4cd3-aefc-38cf0aa6e282)
(Has both the old and new descriptions)

There are likely others I can't spot at a glance

**Reply by Regalis11** [reply]
Yep, this applies to the following talent descriptions:

```
<talentdescription.bootcamp>Gain an additional [xpamount]% experience, until you have obtained [level] talent points.\nWhen you have [level] or more talent points, gain +[skillamount] [skilltype] instead.</talentdescription.bootcamp>
<talentdescription.protectandserve>Gain [xpbonus]% bonus XP and [repbonus]% more reputation when completing a [missiontype] mission.</talentdescription.protectandserve>
<talentdescription.camaraderienew>While no crew member has died during a mission, gain [extraexperience]% bonus XP and [resistanceamount]% ‖color:gui.orange‖Physical Damage Resistance‖end‖.</talentdescription.camaraderienew>
<talentdescription.junctionjunkie>Gain one-time rewards for repairing a certain amount of ‖color:gui.orange‖electrical devices‖end‖:</talentdescription.junctionjunkie>
<talentdescription.machinemaniac>Gain one-time rewards for repairing a certain amount of ‖color:gui.orange‖mechanical devices‖end‖:</talentdescription.machinemaniac>
<talentdescription.warstories>At the end of each round, if you killed any enemies, create a [book] in your inventory. Reading it gives [skill] skill to the reader; the skill gain is determined by the amount of enemies killed during the round.</talentdescription.warstories>
<talentdescription.nobodyimportantdies>Medical items you apply are +[amount]% more potent when applied to non-assistants. Treating assistants is [penalty]% less effective.</talentdescription.nobodyimportantdies>
<talentdescription.healthinsurance>If your health has dropped below [amount]% during a mission, gain [money] mk.</talentdescription.healthinsurance>
```

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/14b11e86754a23dc96f2d1bd687dc8a1425fe3d3 (commit in our private repo), hopefully coming up in the next hotfix!
